docs/software/sciapps/namd.md

```diff
@@ -1,4 +1,274 @@
 [](){#uenv-namd}
 # NAMD
-!!! todo
-    complete docs
+
+[NAMD] is a parallel molecular dynamics code based on [Charm++], designed for high-performance simulations of large biomolecular systems.
+
+!!! danger "Licensing Terms and Conditions"
+    [NAMD] is distributed free of charge for research purposes only and not for commercial use: users must agree to [NAMD license] in order to use it at [CSCS]. Users agree to acknowledge use of [NAMD] in any reports or publications of results obtained with the Software (see [NAMD Homepage] for details).
+
+[NAMD] is provided in two flavours on [CSCS] systems:
+
+* Single-node build
+* Multi-node build
+
+The single-node build works on a single node and benefits from the new GPU-resident mode (see [NAMD 3.0b6 GPU-Resident benchmarking results] for more details).
+The multi-node build works on multiple nodes and is based on [Charm++] MPI backend.
+
+!!! note "Prefer the single-node build and exploit GPU-resident mode"
+    Unless you have good reasons to use the multi-node build, we recommend using the single-node build with the GPU-resident mode.
+
+=== "Single-node build"
+
+    The single-node build provides the following views:
+
+    * `namd-single-node` (standard view, with NAMD)
+    * `develop-single-node` (development view, without NAMD)
+
+    ## Running NAMD 
+
+    The following SBATCH script shows how to run NAMD on a single node with 4 GPUs:
```

Comment:
```suggestion
    The following sbatch script shows how to run NAMD on a single node with 4 GPUs:
```
? I _think_ that's the preferred capitalization, but we don't have many examples yet in the docs (the ones that are there are lower case).

Reply:
Good point, it is lowercase also in the Slurm documentation, so we should stick to that.